src/components/CampaignForm.tsx

```diff
@@ -48,18 +86,46 @@ const CreateCampaignForm = ({ onSubmit, onCancel }: CreateCampaignFormProps): JS
             ),
             size: 12,
         },
+        {
+            field: (
+                <Autocomplete
+                    defaultValue={{ first_name:campaign?.manager?.name.split(' ')[0], id: campaign?.manager?.id, last_name: campaign?.manager?.name.split(' ')[1] } as Partial<ZetkinPerson>  || null}
+                    getOptionLabel={ person => `${person.first_name || ''} ${person.last_name || ''}` }
+                    getOptionValue={ person => person.id || 0 }
+                    label={ intl.formatMessage({ id: 'misc.formDialog.campaign.manager' }) }
+                    name="manager_id"
+                    onInputChange={ (_, v) => {
+                        setSearchFieldValue(v);
+                    } }
+                    options={ people }
```

Comment:
When `people` is empty (before the first search) the label in the options list is "No options". If possible, it would be nice to have some sort of hint there that says "Start typing to search".

Also, it would be nice to display a progress/loading indicator while searching.